### ISSUE TITLE
452: out of range pointer: 0x7feb41e0

### ISSUE THREAD

**Issue by ranjithsnair** [emmett1/lfs-scripts#42]
Getting and error "452: out of range pointer: 0x7feb41e0" after selecting the OS from GRUB screen. I am using Oracle VirtualBox to boot my iso

**Comment by emmett1** [comment]
Whats your host OS for making the iso?

**Comment by ranjithsnair** [comment]
> Whats your host OS for making the iso?

I am building LFS iso from an ubuntu machine hosted in azure cloud. Actually i tried replacing my linux kernel with your kernel and it started showing the logs so i guess my kernel is the problem. Is there any specific settings need to be enabled to boot kernel using EFI ? I mean in menuconfig during kernel build. Legacy boot is working fine with isolinux. My kernel version is 6.4.12

**Comment by ranjithsnair** [comment]
yes, you are right, when i build it from LFS chroot all working good, Also i rebuild the kernel with all the configuration from below URL

https://www.linuxfromscratch.org/blfs/view/12.0/postlfs/grub-setup.html
